### ISSUE TITLE
Allow to be enabled/disabled per course

### ISSUE THREAD

**Issue by Koslas1** [powerus117/RLAfkMarksCanafis#7]
I really like this plugin, and like how you've expanded it to all courses. However, as I can only boost for the ardougne course, I don't don't want to waste my boost waiting for marks of grace, or when going for ape atoll agility laps to get the collection log slots. I would love to be able to have an option to disable the plugin for each course in the plugin options. Such as a tickbox next to each course for if it's enabled (And maybe if possible to have it in a dropdown section which is hidden by default)
That way I would be able to disable this per course, such as when getting laps for ape atoll monkey collection logs, and until I no longer have to boost for Ardougne course, and won't have to turn off the plugin, and then remember to re enable it

**Comment by TomBMNA** [comment]
When talking about enabling or disabling it, you specifically mean the feature where the last obstacle is not clickable I suppose?

**Comment by Koslas1** [comment]
What I'm meaning is having a checkbox next to every course, ticked by default, and if I were to untick a course, for example ape atoll, carry on making the plugin work the exact same way, except if I get a mark of grace on the ape atoll course, don't do anything such as starting a timer, or making the last obstacle right click, but then if I go do agility at a course that I leave ticked, then do what the plugin is meant to do.
So basically allowing you to choose if this plugin is enabled/disabled per course, and then being able to leave the plugin turned on all the time, rather than having to turn off the plugin if I do a course that I don't want plugin to work on.
If that makes sense 😅

**Comment by powerus117** [comment]
Yes it makes sense, but the plugin generally only gives info, and I don't see any harm in showning the info. The only thing actually impacting gameplay is making the last obstacle right click. So I'm just confirming that only affecting this feature would be enough for your request
